amethyst_animation/src/util.rs

```diff
@@ -20,7 +20,10 @@ where
     I: Send + Sync + 'static,
     T: AnimationSampling,
 {
-    controls.entry(entity).unwrap().or_insert_with(AnimationControlSet::default)
+    controls
+        .entry(entity)
+        .unwrap()
```

Comment:
This should not unwrap but return the error.

Reply:
That's a known (and reported issue), yes, will fix that in a separate PR as it changes the API.